Editor/Analytics/AmplitudeEditorLogger.cs

```diff
@@ -59,190 +61,201 @@ public void LogOpenProject()
             {
-                { Constants.Properties.CACHING_ENABLED, isCachingEnabled }
+                { Properties.CACHING_ENABLED, isCachingEnabled }
             });
         }
 
         public void LogClearLocalCache()
         {
-            LogEvent(Constants.EventName.CLEAR_LOCAL_CACHE);
+            LogEvent(EventName.CLEAR_LOCAL_CACHE);
         }
 
         public void LogViewPrivacyPolicy()
         {
-            LogEvent(Constants.EventName.PRIVACY_POLICY);
+            LogEvent(EventName.PRIVACY_POLICY);
         }
 
         public void LogShowInExplorer()
         {
-            LogEvent(Constants.EventName.SHOW_IN_EXPLORER);
+            LogEvent(EventName.SHOW_IN_EXPLORER);
         }
 
         public void LogFindOutMore(HelpSubject subject)
         {
-            LogEvent(Constants.EventName.FIND_OUT_MORE, new Dictionary<string, object>
+            LogEvent(EventName.FIND_OUT_MORE, new Dictionary<string, object>
             {
-                { Constants.Properties.CONTEXT, helpDataMap[subject] }
+                { Properties.CONTEXT, helpDataMap[subject] }
             });
         }
 
         public void LogOpenSetupGuide()
         {
-            LogEvent(Constants.EventName.OPEN_SETUP_GUIDE);
+            LogEvent(EventName.OPEN_SETUP_GUIDE);
         }
 
         public void LogOpenIntegrationGuide()
         {
-            LogEvent(Constants.EventName.OPEN_INTEGRATION_GUIDE);
+            LogEvent(EventName.OPEN_INTEGRATION_GUIDE);
         }
 
         public void LogLoadQuickStartScene()
         {
-            LogEvent(Constants.EventName.LOAD_QUICK_START_SCENE);
+            LogEvent(EventName.LOAD_QUICK_START_SCENE);
         }
 
         public void LogOpenAvatarDocumentation()
         {
-            LogEvent(Constants.EventName.OPEN_AVATAR_DOCUMENTATION);
+            LogEvent(EventName.OPEN_AVATAR_DOCUMENTATION);
         }
 
         public void LogOpenAnimationDocumentation()
         {
-            LogEvent(Constants.EventName.OPEN_ANIMATION_DOCUMENTATION);
+            LogEvent(EventName.OPEN_ANIMATION_DOCUMENTATION);
         }
 
         public void LogOpenAvatarCreatorDocumentation()
         {
-            LogEvent(Constants.EventName.OPEN_AVATAR_CREATOR_DOCUMENTATION);
+            LogEvent(EventName.OPEN_AVATAR_CREATOR_DOCUMENTATION);
         }
 
         public void LogOpenOptimizationDocumentation()
         {
-            LogEvent(Constants.EventName.OPEN_OPTIMIZATION_DOCUMENTATION);
+            LogEvent(EventName.OPEN_OPTIMIZATION_DOCUMENTATION);
         }
 
         public void LogAvatarCreatorSampleImported()
         {
-           LogEvent(Constants.EventName.AVATAR_CREATOR_SAMPLE_IMPORTED);
+            LogEvent(EventName.AVATAR_CREATOR_SAMPLE_IMPORTED);
         }
-        
+
+        public void LogPackageInstalled(PackageCoreInfo packageInfo)
+        {
+            LogEvent(EventName.INSTALL_PACKAGE, new Dictionary<string, object>
```

Comment:
you could perhaps instead add a PackageInfo.ToDictionary function instead of needing to create a new one here? Only thing you might need to do is make sure of correct capitalization. Just an idea though :D

Reply:
Interesting idea! I sort of like it as it is, since it's explicit to the method and you can clear look here to see exactly what properties are being sent to Amplitude. Maybe worth exploring though 🤔